src/fonduer/learning/classifier.py

```diff
@@ -4,23 +4,49 @@
 
 
 class Classifier(object):
-    """An abstract class for a probabilistic classifier."""
+    """An abstract class for a probabilistic classifier.
+
+    :param cardinality: Cardinality of class
+    :type cardinality: int
+    :param name: Name of the model
+    :type name: str
+    """
 
     def __init__(self, cardinality=2, name=None):
         self.name = name or self.__class__.__name__
         self.cardinality = cardinality
 
     def marginals(self, X, batch_size=None, **kwargs):
```

Comment:
Are kwargs needed here? Can we enumerate what they are hiding?

Reply:
Done!